lib/adhearsion/call_controller/dial.rb

```diff
@@ -112,7 +112,11 @@ def run(controller)
           track_originating_call
           start_ringback controller
           prep_calls
-          place_calls
+          begin
+            place_calls
+          rescue Adhearsion::Call::Hangup => e
+            logger.info "#{self.class}.#{__method__}:#{__LINE__}: Party Hangup - #{e.inspect}\n  #{(e.backtrace || ['EMPTY BACKTRACE']).join("\n  ")}"
+          end
```

Comment:
The function of this rescue is to absorb an exception potentially raised by prep_calls/run, effectively making the client application unaware that a Hangup occurred.  Personally, I don't think we should rescue at all here.  What do you think?

Reply:
i hear you, and agree this may very likely not be in the right spot.  my chief desire here, is based on the fact that this is happening during the call to `run` that occurs in `dial/dial_and_confirm`, and that i wanted those callers to proceed with the `await_completion`, `cleanup_calls` etc. that happen after the call to `run` -  `cleanup_calls` being the most important one i think, b/c it's in charge of making sure all the outbound calls get hungup.  i _believe_ that a hangup of those outbound calls would raise the Hangup exception, and so we'd maintain that _a_ hangup event occurred.

i'm going to revisit this some more regardless, and get a bit more certain about what i'm trying to say or whether we simply need to handle this differently :)